### ISSUE TITLE
extension_access_modifier shouldn't trigger on open declarations

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#1629]
```swift
// shouldn't trigger
extension Foo {
    open override func bar() {
    }
}
```
